### DISCUSSION TITLE
Detecting Packages in Package Box

### DISCUSSION THREAD

**Discussion by CraigHaynie** [blakeblackshear/frigate#4732] [Ask A Question]
First of all, I am LOVING Frigate! 

I have several cameras including a package box. I would like to have an event when something in placed in the box. It could be that it detects motion, or sees light, or detects a box I suppose. I am going to be playing with the motion event in HomeAssistant as an option but wondered if anyone has determined a way to handle this natively in Frigate? 

![image](https://user-images.githubusercontent.com/64322632/208317522-3af6cac1-d234-4126-b238-6c82e66b0491.png)

**Comment by NickM-27** [comment]
Interesting idea, I honestly think the best way would just be a manual event via api call along with a door sensor or something, but that's not implemented fully yet https://github.com/blakeblackshear/frigate/pull/3184

For now motion or maybe person when their hand places it in.

**Reply by CraigHaynie** [reply]
I am a little lost - are you suggesting that I can trigger an event (that would record and show up as an event) from an API call to Frigate (from Home Assistant)? If so, couldn't I detect motion in HA and then tell Frigate it is an event? If this is what you are saying, are you aware of an example of where someone has done this? 

When you say "for now" motion or person, what do you mean? Frigate doesn't seem to have the ability to trigger an event on motion. And, a hand is likely not enough

**Reply by NickM-27** [reply]
> couldn't I detect motion in HA and then tell Frigate it is an event? If this is what you are saying, are you aware of an example of where someone has done this?

this is what I am suggesting however at the end my point was it is currently not supported (I have a draft PR but it is not merged)

>  Frigate doesn't seem to have the ability to trigger an event on motion

There is an HA sensor in frigate for when motion is detected, so could take a snapshot, send a notification, etc. is what I mean

> And, a hand is likely not enough

The default model is from the coco dataset which has quite a few examples of hands so with a lowered min_score I could see it being worth a try

**Reply by CraigHaynie** [reply]
I will admit I am a little lost with what you are saying. I WANT to understand but I just don't lol. If you think of a "tutotial" or if something comes up and you remember me please do let me know!  : )
